src/data/api.js

```diff
@@ -52,7 +51,21 @@ export async function fetchLatestAttempt(courseId) {
 }
 
 export async function pollExamAttempt(url) {
-  const { data } = await getAuthenticatedHttpClient().get(`${getConfig().LMS_BASE_URL}${url}`);
+  let data;
+  if (!getConfig().EXAMS_BASE_URL) {
+    const edxProctoringURL = new URL(
+      `${getConfig().LMS_BASE_URL}${url}`,
+    );
+    const urlResponse = await getAuthenticatedHttpClient().get(edxProctoringURL.href);
+    data = urlResponse.data;
+  } else {
+    data = await fetchActiveAttempt();
+    data = data.attempt;
+
+    // Update dictionary to have correct status key for legacy compatibility
+    Object.defineProperty(data, 'status', Object.getOwnPropertyDescriptor(data, 'attempt_status'));
```

Comment:
the data is just a dictionary, so does it need to be accessed like this? Or could you just say `data.status = data.attempt_status`? 

Also how does this work when edx-exams passes the request to edx-proctoring? edx-proctoring will still still return `status`, so it is probably worth checking if that key already exists in the data.

Reply:
Good point. I'm going to make that change now.